pages/notifications.js

```diff
@@ -136,33 +178,46 @@ const Page = withT(({ url: { query, query: { context, token, tokenType, noAutoAu
     }
   ].filter(Boolean)
 
-  const logo = context === 'projectr' ? (
+  const isProjectR = context === 'projectr'
+  const logo = isProjectR ? (
     <a href='https://project-r.construction/' rel='noopener' target='_blank' {...styles.logoProjectR}>
       <img
         style={{height: 50}}
         src={`${CDN_FRONTEND_BASE_URL}/static/project_r_logo.png`} />
     </a>
   ) : (
     hasCurtain
-      ? <Logo height={34} />
-      : <a href='/' target={logoTarget}>
-        <Logo height={34} />
+      ? <div {...styles.logoRepublik}>
+        <Logo />
+      </div>
+      : <a href='/' target={logoTarget} {...styles.logoRepublik}>
+        <Logo />
       </a>
   )
 
+  const stickyBar = !isProjectR
+
   return (
     <div>
       <Head>
         <title>{t('notifications/pageTitle')}</title>
         <meta name='robots' content='noindex' />
       </Head>
       <NarrowContainer>
-        {!inNativeApp && (
-          <div {...styles.logo}>
-            {logo}
-          </div>
-        )}
-        <div {...(inNativeApp ? styles.textApp : styles.text)}>
+        <div {...(stickyBar ? styles.bar : undefined)}>
+          {logo}
+        </div>
+        {inNativeApp && <Link route='index'>
+          <a {...styles.close}>
+            <MdClose size={32} fill='#000' />
```

Comment:
Structure: May and shall this color come from SG_* ENV variable?

Reply:
Pure black is always hard coded. Never found a nice abstraction for this. Having a variable `black` which may or may not contain pure black is odd. It's the default fill of the logo and most icons. In this case leaving it out would probably also render black (default).